src/scripts/browser.js

```diff
@@ -122,7 +122,7 @@ define([], function () {
     }
 
     function iOSversion() {
-        if (/iP(hone|od|ad)/.test(navigator.platform)) {
+        if (/iP(hone|od|ad)|MacIntel/.test(navigator.platform)) {
```

Comment:
Maybe add a comment and also fix the crash itself from happening again with a quick check?

Reply:
The check will protect against crushing, but it can shadow the issue: everything works fine, but in fact iOSVersion is undefined, and no one knows about it.
_Once I argued about the same with the boss. :smile: His position was that it is better not to crush during the presentation._